update governance (#81)
hl7ch/ch-term: input/pagecontent/index.md

* [Concept Maps](conceptmaps.html)
* [Naming Systems](namingsystems.html)

### Collaboration & Governance
This implementation guide is the result of collaborative work undertaken with participants from [HL7 Switzerland](https://www.hl7.ch) and [eHealth Suisse](https://www.e-health-suisse.ch/startseite.html).

- A yearly major release is foreseen to support the Swiss balloted IGs, with an interim version published to support the HL7 Switzerland ballot phase.

> - Additions or changes can only be made through reviewed pull requests (PRs). In addition, PRs must be reviewed and accepted by the responsible parties of the respective HL7 Switzerland working groups, if the terminology resources are used within their implementation guides.

Reviewer: Who defines the reviewers for a PR?

Author: Additions or changes can only be made through pull requests (PRs) reviewed by the leader of the working group or the HL7 Switzerland technical manager. In addition .... by the leader of the the respective HL7 Switzerland working groups

> - Patch releases can be made upon request to the FHIR working group. Dependent IGs on CH Term should allow patched versions.

Reviewer: Is it correct, that when there are i.e. additional vaccines to be added to the codesystem, valueset and the corresponding conceptmaps (CH-VACD), a request for patch release has to be requested to the FHIR WG?

* [Concept Maps](conceptmaps.html)
* [Naming Systems](namingsystems.html)

### Collaboration & Governance

This implementation guide is the product of collaborative work undertaken with participants from [HL7 Switzerland](https://www.hl7.ch) and [eHealth Suisse](https://www.e-health-suisse.ch/startseite.html)

> - a yearly major release is foreseen to support the swiss balloted ig's with a interim version published for supporting the Hl7 switzerland ballot phase

Reviewer 2: ```suggestion
- a yearly major release is foreseen to support the swiss balloted ig's with an interim version published for supporting the Hl7 switzerland ballot phase
```

- addition / changes can only be done through reviewed pull requests

> - in addition pull requests (PR) needs to be accepted / reviewed by the responsible of the different swiss HL7.ch work groups, if the terminology resources are used within their implementation guides

Reviewer 2: ```suggestion
- in addition pull requests (PR) need to be accepted / reviewed by the responsible of the different Swiss HL7.ch work groups, if the affected terminology resources are used within their implementation guides
```

- patch releases can be done on request to the FHIR work group, dependent IG's on CH Term should allow patched versions
- for terminology resources where CH Term is not the authority/master, the conformance resource needs to indicate the authority in meta.source ([example for art-decor](https://fhir.ch/ig/ch-term/ValueSet-DocumentEntry.authorRole.json.html))

> - only stable versions of ValueSets will be published if there a multiple versions of ValueSets (e.g. DocumentEntry.typeCode) due that the tooling supports currently one version [background](https://github.com/hl7ch/ch-term/issues/5).

Reviewer 2: ```suggestion
- only stable versions of ValueSets will be published if there a multiple versions of ValueSets (e.g. DocumentEntry.typeCode) due to that the tooling supports currently one version ([background](https://github.com/hl7ch/ch-term/issues/5)).
```
